### DISCUSSION TITLE
What’s supermerills email/Super Slicer's official email?

### DISCUSSION THREAD

**Discussion by hugoToscher** [supermerill/SuperSlicer#3953] [General]
Hi, I need to contact supermerill with some questions on the possibilities of non-planar slicing in Super Slicer. Is there any official contact information of his? Ideally, an email of some sort. @supermerill

**Comment by supermerill** [comment]
For others:
I won't put my good email on internet.
To contact me, you can via discord (put 4TVUG4VRzP after the 'discord.gg', like for any invite link)
or via reddit (supermerill)
